backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
```

Comment:
is `extractMentionedUsers` in the right place `notifications` or should it rather be somewhere else?

Reply:
it’s only used for notifications. Therefore that should be okay. @ulfgebhardt